workflows/ampliseq.nf

```diff
@@ -153,13 +153,13 @@ include { GET_SOFTWARE_VERSIONS         } from '../modules/local/get_software_ve
 
 if (params.pacbio) {
 	//PacBio data
-	cutadapt_options_args       = " --rc -g ${params.FW_primer}...${params.RV_primer}"
+	cutadapt_options_args       = " --rc -g ${params.FW_primer}...${params.RV_primer} --minimum-length 1"
 } else if (params.single_end) {
 	//Illumina SE
-	cutadapt_options_args       = " -g ${params.FW_primer}"
+	cutadapt_options_args       = " -g ${params.FW_primer} --minimum-length 1"
 } else {
 	//Illumina PE
-	cutadapt_options_args       = " -g ${params.FW_primer} -G ${params.RV_primer}"
+	cutadapt_options_args       = " -g ${params.FW_primer} -G ${params.RV_primer} --minimum-length 1"
 }
 
 def cutadapt_options 			= modules['cutadapt']
```

Comment:
I think it would be better to have the `--minimum-length 1` in the `conf/modules.config` similar to all other static parameters, e.g. in this [line](https://github.com/nf-core/ampliseq/blob/adcae8a113d3c0b4c636edcd242614dd846b6d13/conf/modules.config#L30).